e2e/testcafe-devextreme/tests/editors/checkBox/common.ts

```diff
@@ -47,3 +59,104 @@ fixture.disablePageReloads`CheckBox`
     await createWidget('dxCheckBox', { value: undefined, text: 'indeterminate', rtlEnabled: true }, '#indeterminateRTL');
   });
 });
+
+test('Checkbox configuration, different Checkbox icon sizes, states, value Modes', async (t) => {
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+  await testScreenshot(t, takeScreenshot, 'CheckBox configurations different iconSizes.png', { element: '#container', shouldTestInCompact: true });
+
+  await testScreenshot(t, takeScreenshot, 'CheckBox configurations different iconSizes.png', { element: '#container', theme: getDarkThemeName() });
+
+  await t
+    .expect(compareResults.isValid())
+    .ok(compareResults.errorMessages());
+}).before(async () => {
+  for (const iconSize of [20, undefined, 45]) {
+    for (const state of [
+      READONLY_STATE_CLASS,
+      DEFAULT_STATE_CLASS,
+      ACTIVE_STATE_CLASS,
+      HOVER_STATE_CLASS,
+      FOCUSED_STATE_CLASS,
+      DISABLED_STATE_CLASS,
+    ] as string[]
+    ) {
+      for (const value of valueModes) {
+        const id = `dx${new Guid()}`;
+        await appendElementTo('#container', 'div', id, {});
+
+        await createWidget('dxCheckBox', {
+          value,
+          iconSize,
+        }, `#${id}`);
+        await setClassAttribute(Selector(`#${id}`), state);
+      }
+    }
+  }
+});
+
+test('Checkbox configuration, different Checkbox widths', async (t) => {
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+  await testScreenshot(t, takeScreenshot, 'CheckBox configurations different widths.png', { element: '#container', shouldTestInCompact: true });
+
+  await testScreenshot(t, takeScreenshot, 'CheckBox configurations different widths.png', { element: '#container', theme: getDarkThemeName() });
+
+  await t
+    .expect(compareResults.isValid())
+    .ok(compareResults.errorMessages());
+}).before(async () => {
+  for (const width of [45, undefined, 65]) {
+    for (const text of [undefined, 'label', 'one two three']) {
+      for (const state of [
```

Comment:
We test states and value modes in another test. For testing width and label cropping (if that is the intention of this test), we don't need to use several value modes and states. Let's remove them from this test